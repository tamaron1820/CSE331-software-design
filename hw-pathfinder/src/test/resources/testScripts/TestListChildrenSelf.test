# Test whether list of children is existed correctly if there is self edge

CreateGraph graph1

AddNode graph1 n1

AddEdge graph1 n1 n1 2.0

ListChildren graph1 n1