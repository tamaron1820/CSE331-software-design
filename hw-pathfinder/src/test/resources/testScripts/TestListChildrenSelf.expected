# Test whether list of children is existed correctly if there is self edge

created graph graph1

added node n1 to graph1

added edge 2.000 from n1 to n1 in graph1

the children of n1 in graph1 are: n1(2.0)
